Apps/phawswafv2/awswafv2_connector.py

```diff
@@ -454,6 +457,16 @@ def initialize(self):
         # get the asset config
         config = self.get_config()
 
+        self._proxy = {}
+        env_vars = config.get('_reserved_environment_variables', {})
+        if 'HTTP_PROXY' in env_vars:
+            self._proxy['http'] = env_vars['HTTP_PROXY']['value']
+        if 'HTTPS_PROXY' in env_vars:
+            self._proxy['https'] = env_vars['HTTPS_PROXY']['value']
+
+        self._region = AWSWAF_REGION_DICT.get(config[AWSWAF_REGION])
+        self._scope = config.get(AWSWAF_SCOPE)
+
```

Comment:
Do we need to update this part? It looks like we simply read values, and changing order does not seem like it will affect the set values. 🤔

Reply:
I didn't change it, I moved it up in the method so that it happens before the 'use role' section. The way I had it, if 'use role' was checked it would finish the initialization and return before setting the region.

Reply:
So to answer your question, yes, it only reads values but the order does affect the outcome.

Reply:
ah I see. Thank you for the explanation. 👍